src/DataManagement/Saving/CalculationResult.cs

```diff
@@ -0,0 +1,26 @@
+﻿using ePiggyWeb.Utilities;
+using System.Collections.Generic;
+
+namespace ePiggyWeb.DataManagement.Saving
+{
+    public class CalculationResults
+    {
+        public IList<ISavingSuggestion> EntrySuggestions;
+        public List<SavingSuggestionByMonth> MonthlySuggestions;
+        public int TimesToRepeatSaving;
+
+        public CalculationResults(IList<ISavingSuggestion> entrySuggestions, List<SavingSuggestionByMonth> monthlySuggestions, int timesToRepeatSaving)
+        {
+            EntrySuggestions = entrySuggestions;
+            MonthlySuggestions = monthlySuggestions;
+            TimesToRepeatSaving = timesToRepeatSaving;
+        }
+
+        public CalculationResults()
+        {
+            EntrySuggestions = new List<ISavingSuggestion>();
+            MonthlySuggestions = new List<SavingSuggestionByMonth>();
+            TimesToRepeatSaving = 0;
+        }
```

Comment:
Where do you see empty constructor used? Since if it is empty there's really no point in displaying it etc.

Reply:
I think I used it in the SendData method in the front, as a quick work-around to test with compiling, or something, though it doesn't seem to have references anymore, so in such a case should i just delete it?

Reply:
I think best practice would be to delete it